satpy/composites/__init__.py

```diff
@@ -1410,6 +1416,9 @@ def __init__(self, name, filename=None, area=None, **kwargs):
     def __call__(self, *args, **kwargs):
         """Call the compositor."""
         from satpy import Scene
+        # Check if filename exists, if not then try from SATPY_ANCPATH
+        if (not os.path.isfile(self.filename)):
+            self.filename = os.path.join(get_environ_ancpath(), self.filename)
```

Comment:
I was almost suggesting that maybe we could add a downloader and save to `$SATPY_ANCPATH`, but looking at `satpy/config.py` the default is within the `satpy` installation, so maybe that's not a good choice. Something what `pyspectral` does with the RSR files would work nicely, but I'm fine how this is now :-)

Reply:
Could you make it so it only changes the filename *if* the ancpath version of the file exists too? Otherwise future error messages may be confusing (path won't match what the user provided).